Airports/EGGW/Stars.txt

```diff
@@ -1,31 +1,49 @@
-STAR:EGGW:07:FINMA1L:FINMA BOMBO BKY BUSTA LOREL
-STAR:EGGW:25:LISTO1L:LISTO PEDIG ROGBI FINMA BOMBO BKY BUSTA LOREL
-STAR:EGGW:07:LISTO1L:LISTO PEDIG ROGBI FINMA BOMBO BKY BUSTA LOREL
+STAR:EGGW:25:FINMA1N:FINMA WOBUN EDCOX JUMZI ZAGZO
+STAR:EGGW:07:FINMA1N:FINMA WOBUN EDCOX JUMZI ZAGZO
+STAR:EGGW:25:LISTO1L:LISTO PEDIG ROGBI FINMA WOBUN EDCOX JUMZI ZAGZO
+STAR:EGGW:07:LISTO1L:LISTO PEDIG ROGBI FINMA WOBUN EDCOX JUMZI ZAGZO
 STAR:EGGW:25:LOGAN2A:LOGAN CLN ABBOT
 STAR:EGGW:07:LOGAN2A:LOGAN CLN ABBOT
-STAR:EGGW:25:RINIS1A:RINIS IDESI LAPRA ABBOT
-STAR:EGGW:07:RINIS1A:RINIS IDESI LAPRA ABBOT
-STAR:EGGW:25:SILVA1L:SILVA BOMBO BKY BUSTA
-STAR:EGGW:07:SILVA1L:SILVA BOMBO BKY BUSTA
-STAR:EGGW:25:TELTU1L:TELTU VATON BPK BKY BUSTA LOREL
-STAR:EGGW:07:TELTU1L:TELTU VATON BPK BKY BUSTA LOREL
-STAR:EGGW:25:TOSVA1A:TOSVA IDESI LAPRA ABBOT
-STAR:EGGW:07:TOSVA1A:TOSVA IDESI LAPRA ABBOT
-STAR:EGGW:25:XAMAN1A:XAMAN IDESI LAPRA ABBOT
-STAR:EGGW:07:XAMAN1A:XAMAN IDESI LAPRA ABBOT
+STAR:EGGW:25:RINIS1N:RINIS IDESI WIQID MUCTE OFJES UDDIZ COCCU JUMZI ZAGZO
+STAR:EGGW:07:RINIS1N:RINIS IDESI WIQID MUCTE OFJES UDDIZ COCCU JUMZI ZAGZO
+STAR:EGGW:25:SILVA1N:SILVA WOBUN EDCOX JUMZI ZAGZO
+STAR:EGGW:07:SILVA1N:SILVA WOBUN EDCOX JUMZI ZAGZO
+STAR:EGGW:25:TELTU1N:TELTU MOREZ VATON OZZOT BPK ILLOC OXDUF COCCU JUMZI ZAGZO
+STAR:EGGW:07:TELTU1N:TELTU MOREZ VATON OZZOT BPK ILLOC OXDUF COCCU JUMZI ZAGZO
+STAR:EGGW:25:UNDUG1N:UNDUG MAY VATON OZZOT BPK ILLOC OXDUF COCCU JUMZI ZAGZO
+STAR:EGGW:07:UNDUG1N:UNDUG MAY VATON OZZOT BPK ILLOC OXDUF COCCU JUMZI ZAGZO
+STAR:EGGW:25:XAMAN1N:XAMAN IDESI WIQID MUCTE OFJES UDDIZ COCCU JUMZI ZAGZO
+STAR:EGGW:07:XAMAN1N:XAMAN IDESI WIQID MUCTE OFJES UDDIZ COCCU JUMZI ZAGZO
+
+;Withdrawn 2022/02
+STAR:EGGW:25:oldAVANT1L:AVANT OCK VATON BPK BKY BUSTA LOREL
+STAR:EGGW:07:oldAVANT1L:AVANT OCK VATON BPK BKY BUSTA LOREL
+STAR:EGGW:25:oldBANVA1L:BANVA UNDUG MAY VATON BPK BKY BUSTA LOREL
+STAR:EGGW:07:oldBANVA1L:BANVA UNDUG MAY VATON BPK BKY BUSTA LOREL
+STAR:EGGW:25:oldBARMI2A:BARMI MEGEL DITOB LAPRA ABBOT
+STAR:EGGW:07:oldBARMI2A:BARMI MEGEL DITOB LAPRA ABBOT
+STAR:EGGW:25:oldBEDEK1L:BEDEK NIGIT OCK VATON BPK BKY BUSTA LOREL
+STAR:EGGW:07:oldBEDEK1L:BEDEK NIGIT OCK VATON BPK BKY BUSTA LOREL
+STAR:EGGW:25:oldFINMA1L:FINMA BOMBO BKY BUSTA LOREL
+STAR:EGGW:07:oldFINMA1L:FINMA BOMBO BKY BUSTA LOREL
+STAR:EGGW:25:oldLISTO1L:LISTO PEDIG ROGBI FINMA BOMBO BKY BUSTA LOREL
+STAR:EGGW:07:oldLISTO1L:LISTO PEDIG ROGBI FINMA BOMBO BKY BUSTA LOREL
+STAR:EGGW:25:oldRINIS1A:RINIS IDESI LAPRA ABBOT
+STAR:EGGW:07:oldRINIS1A:RINIS IDESI LAPRA ABBOT
+STAR:EGGW:25:oldSILVA1L:SILVA BOMBO BKY BUSTA
+STAR:EGGW:07:oldSILVA1L:SILVA BOMBO BKY BUSTA
+STAR:EGGW:25:oldTELTU1L:TELTU VATON BPK BKY BUSTA LOREL
+STAR:EGGW:07:oldTELTU1L:TELTU VATON BPK BKY BUSTA LOREL
+STAR:EGGW:25:oldTOSVA1A:TOSVA IDESI LAPRA ABBOT
+STAR:EGGW:07:oldTOSVA1A:TOSVA IDESI LAPRA ABBOT
+STAR:EGGW:25:oldXAMAN1A:XAMAN IDESI LAPRA ABBOT
+STAR:EGGW:07:oldXAMAN1A:XAMAN IDESI LAPRA ABBOT
 
 ;Withdrawn 2020/13
```

Comment:
Can remove?

Reply:
No, these are only just going this AIRAC. TOSVA1A, XAMAN1A are routes people with incorrect AIRACs might still have to fly.

Reply:
The 2020/13 ones can be.